src/App.tsx

```diff
@@ -0,0 +1,97 @@
+// Libs
+import * as React from "react";
+import { Grid, ButtonGroup, Button } from "@material-ui/core";
+
+// Components
+import Charts from "./components/Charts/Charts";
+import Grids from "./components/Grids/Grids";
+import Transactions from "./components/Transactions/Transactions";
+
+// Assets
+import { iTransaction, category } from "./types";
+import "./App.css";
+
+export default function App() {
+  // Definitions
+  const [cleanTransactions, setCleanTransactions] = React.useState<
+    iTransaction[]
+  >([]);
+
+  const tabs = {
+    0: <Charts></Charts>,
```

Comment:
donne leur des noms plutôt que des numéros, c'est plus lisible!

Reply:
Si les éléments sont vides (pas d'enfants) tu peux les écrire `<Charts />`

Reply:
Added to backlog: rework structure configuration.